package.json

```diff
@@ -0,0 +1,137 @@
+    "dom-serializer": "^1.4.1",
+    "domelementtype": "^2.3.0",
+    "domhandler": "^4.3.1",
+    "domutils": "^2.8.0",
+    "dot-case": "^3.0.4",
+    "electron-to-chromium": "^1.4.284",
+    "enhanced-resolve": "^5.12.0",
+    "entities": "^2.2.0",
+    "envinfo": "^7.8.1",
+    "es-module-lexer": "^0.9.3",
+    "escalade": "^3.1.1",
+    "eslint-scope": "^5.1.1",
+    "esrecurse": "^4.3.0",
+    "estraverse": "^4.3.0",
+    "events": "^3.3.0",
+    "fast-deep-equal": "^3.1.3",
+    "fast-json-stable-stringify": "^2.1.0",
+    "fastest-levenshtein": "^1.0.16",
+    "find-up": "^4.1.0",
+    "function-bind": "^1.1.1",
+    "glob-to-regexp": "^0.4.1",
+    "graceful-fs": "^4.2.10",
+    "has": "^1.0.3",
+    "has-flag": "^4.0.0",
+    "he": "^1.2.0",
+    "html-minifier-terser": "^6.1.0",
+    "html-webpack-plugin": "^5.5.0",
+    "htmlparser2": "^6.1.0",
+    "import-local": "^3.1.0",
+    "interpret": "^3.1.1",
+    "is-core-module": "^2.11.0",
+    "is-plain-object": "^2.0.4",
+    "isexe": "^2.0.0",
+    "isobject": "^3.0.1",
+    "jest-worker": "^27.5.1",
+    "json-parse-even-better-errors": "^2.3.1",
+    "json-schema-traverse": "^0.4.1",
+    "kind-of": "^6.0.3",
+    "loader-runner": "^4.3.0",
+    "locate-path": "^5.0.0",
+    "lodash": "^4.17.21",
+    "lower-case": "^2.0.2",
+    "merge-stream": "^2.0.0",
+    "mime-db": "^1.52.0",
+    "mime-types": "^2.1.35",
+    "neo-async": "^2.6.2",
+    "no-case": "^3.0.4",
+    "node-releases": "^2.0.8",
+    "nth-check": "^2.1.1",
+    "p-limit": "^2.3.0",
+    "p-locate": "^4.1.0",
+    "p-try": "^2.2.0",
+    "param-case": "^3.0.4",
+    "pascal-case": "^3.1.2",
+    "path-exists": "^4.0.0",
+    "path-key": "^3.1.1",
+    "path-parse": "^1.0.7",
+    "picocolors": "^1.0.0",
+    "pkg-dir": "^4.2.0",
+    "pretty-error": "^4.0.0",
+    "punycode": "^2.1.1",
+    "randombytes": "^2.1.0",
+    "rechoir": "^0.8.0",
+    "relateurl": "^0.2.7",
+    "renderkid": "^3.0.0",
+    "resolve": "^1.22.1",
+    "resolve-cwd": "^3.0.0",
+    "resolve-from": "^5.0.0",
+    "safe-buffer": "^5.2.1",
+    "schema-utils": "^3.1.1",
+    "serialize-javascript": "^6.0.0",
+    "shallow-clone": "^3.0.1",
+    "shebang-command": "^2.0.0",
+    "shebang-regex": "^3.0.0",
+    "source-map": "^0.6.1",
+    "source-map-support": "^0.5.21",
+    "strip-ansi": "^6.0.1",
+    "style-loader": "^3.3.1",
+    "supports-color": "^8.1.1",
+    "supports-preserve-symlinks-flag": "^1.0.0",
+    "tapable": "^2.2.1",
+    "terser": "^5.16.1",
+    "terser-webpack-plugin": "^5.3.6",
+    "tslib": "^2.4.1",
+    "update-browserslist-db": "^1.0.10",
+    "uri-js": "^4.4.1",
+    "utila": "^0.4.0",
+    "watchpack": "^2.4.0",
+    "webpack": "^5.75.0",
+    "webpack-cli": "^5.0.1",
+    "webpack-merge": "^5.8.0",
+    "webpack-sources": "^3.2.3",
+    "which": "^2.0.2",
+    "wildcard": "^2.0.0"
+  },
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "webpack"
+  },
+  "repository": {
```

Comment:
- [ ] Please make sure to add a `development server` script to start the server. Currently, I'm not able to start the server. Please check your webpack configurations.(Weppack server should be started with CLI, It should start with with ex `npm run start`, `npm run dev`).